upstream-community-operators/enc-key-sync-operator/0.0.1/enc-key-sync-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,339 @@
+       --type=key \
+       --from-file=my-priv-key.pem \
+       my-decryption-key
+     ```
+
+    In this case, we installed the operator in namespace `enc-key-sync`,
+    and have a private key locally with filename `my-priv-key.pem`. If you are
+    following this example, you can use the following private key:
+
+      ```
+      $ cat > my-priv-key.pem <<EOF
+      -----BEGIN RSA PRIVATE KEY-----
+      MIIEpAIBAAKCAQEAv8Ny7dCWQ8Pdq1ddYSwkQOCB3lUEZVEyj9StX3jnISF/rxIs
+      UZzJfbOrQN0fGkm+1sCCtltgQdztTjito8FhDGflqQBSmV40XP3iZnNUJDrHuAol
+      463Z/BuxxFXL3ry6rTosLGfrRwdQjxp8RSsnWyIIO2rmcqXZYe4SCtiMjMejLlTI
+      DWLIMdYL3d6hA4DpgDLoh6EPmhKMVVwRt5b0ew5eMLcDuq6ButOM5yv4zYVHNraj
+      Y41NK+abSlFb6wzMg2AUDiC/MxV1LRq6mpyZGJllx3LS1M1j7fDO3pmh/M0X7yD/
+      4RgHwFaW4/4CQBw3fyxrOv0pZzZay+o/pyMMWwIDAQABAoIBAQC8HV2HEZH21BOG
+      W+PMyWKfCh4cHsZ7JZY2JmoVOCN0CsqY0XkMboPyfehHbyNtxF4jiSIxBZ59vd5F
+      V7Bu7eroIpvWl+xva0xu1NfdrNEj4U2+qqXUnd0zRW+zrH6b+AQgnupqfV7+hJxw
+      ZYj2yYiIC/CLaSi72xpOyR6F6TyndBVRFePoktfCsPcevQDTVeYEbTr9q52vwK3d
+      nROr3bV+J2y5Glce/4A6yTknJoDNWcDFvy3Ai//5bpV2x38E+FNRRb1BfZN56/7M
+      3bEvwLyJ2bvnEqAQ+7Z8+307aMiMr01s1IQLYvi5Z1fvNewoaj1yxjxfawPvcMtB
+      YDg6soDRAoGBAOA8zv/a48WD+dCb9WFk4YtRqg3ZjwSRKT3Afsfkd5lGz8yRlGP4
+      HXCP8nQ7c2TUEjqiQmtyxrq7yi5dacIzzZXAlS8ORA5BxCfrxzUBvuPKgGoxeozC
+      /Xef7mJ6Si+JDIjY2GKwO8I66JgxScTi7JWlyXmQ26rY4/0Yg8f6Au8XAoGBANrt
+      FPYVXn4S6VdAdhFzkx85ymQ+uX6yflRIUcmP5sySKMCPD1GSrVujDIgj485sEmH2
+      h57gDWzFWLmEU6PDsG2Bpsi29w5MMibr4y7Ez6y1rNd9eor5lDIhKi2sJRA0Ftj1
+      tBl31rhkJfCnzVIRn3Q+ZuvgTca7J9oUnCFBlfddAoGAUih1f3Dnu1qbkT9TLJgV
+      u0H0mJZ5vCajgaihywN+fn5fbIh6YhZqUu+q2cNeiDbbZvhEdbHb9lcPwOUg9rKc
+      RJ4HCvKjJMYb5LSSjG1TT4rGeiIe0Kwwyj+izBoaTEhee1VYEvCXNJb42apVaPnr
+      zPitVQkqMvK8teLhhceog4kCgYA2M/C2pL/KcyA2rA0PcRAB8Sr8+tKuXb8NWwJ0
+      5x37lExmsITYa3pkb9AQfOJQH03F12Xongx027+F3w9eQnsSAcGrfDFa5t6b6FdN
+      IwlP94Mdr0GB2x0n9DIfMLnUczEc8mhuzc7pxFHobYNWSGq0Oyb8S4K2K2xIgEXP
+      rg9VOQKBgQCvZXPe7uRyfeFxH9uexhPOZEYNpF8/SxeFnTmxoyDNgfTaT6wC0Vwt
+      jALXNbZLiYen9cUBfusELk8chLna1tCvLDUTT0m/Y80d8p+S80EsZ20ja1A3HdBv
+      h1TW/ep4aPeI0UE0ZNOifUB37IGXETO5fohbzm799dH8jtAkE02pww==
+      -----END RSA PRIVATE KEY-----
+      EOF
+      ```
+
+    ### Running an encrypted container
+
+    Great - we are done setting up the operator and configuring keys, now we can
+    run encrypted container images! If you followed along with the provided
+    private key, you should be able to run the image `docker.io/lumjjb/sample-enc-app`
+
+     ```
+     $ kubectl run enc-workload --image=docker.io/lumjjb/sample-enc-app
+     ```
+
+    Great you have successfully run an encrypted container image!
+
+
+  maturity: alpha
+  keywords:
+    - encryption
+    - decryption
+    - oci
+    - image
+    - container
+  maintainers:
+    - name: Brandon Lum
+      email: lumjjb@gmail.com
+  provider:
+    name: IBM
+  version: 0.0.1
+  replaces: ''
```

Comment:
Please remove replaces, since it is the first version of operator and use it only when updating to newer version

Reply:
Removed this line and repushed (squashed)! Tks